recipe/meta.yaml

```diff
@@ -11,35 +11,37 @@ package:
 
 source:
   fn: {{ name }}-{{ version }}.tar.xz
-  url: http://ftp.gnome.org/pub/GNOME/sources/pygobject/{{ majmin }}/{{ name }}-{{ version }}.tar.xz
+  url: https://download.gnome.org/sources/{{ name }}/{{ majmin }}/{{ name }}-{{ version }}.tar.xz
   sha256: {{ sha256 }}
   patches:
     - macos-tests.patch
     - meson-rpaths.patch  # [linux]
+    - meson-tests-long-filename.patch
 
 build:
-  number: 1
+  number: 0
   skip: true  # [py<35]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - m2-msys2-runtime  # [win]
     - ninja
-    - {{posix}}patch
-    - {{native}}pkg-config
+    - {{  posix  }}patch
+    - {{  native  }}pkg-config
   host:
     - flake8  # for testing during the build
     - glib
-    - gobject-introspection 1.*
+    - gobject-introspection
+    - gtk3  # for testing during the build
```

Comment:
I believe that, by default, adding this dep will propagate a runtime dependency on `gtk3` into the generated packages. We don't want that to occur, and we should be careful to check that it doesn't happen.